test/core/Do.Test.Architecture/ArchitectureSpecExtensions.cs

```diff
@@ -108,8 +109,8 @@ public static void ShouldNotHave<T>(this ApplicationContext context, T value)
         phases ??= new[] { phase ?? mockMe.APhase() };
 
         var result = new Mock<ILayer>();
-
         result.Setup(l => l.GetPhases()).Returns(phases);
+        result.Setup(l => l.Id).Returns(id ?? string.Empty);

```

Comment:
make it `$"{Guid.NewGuid()}"` instead of `string.Empty` to avoid unnecessary ids in above tests